### PR TITLE
Followup PR for https://github.com/sylabs/sif/pull/45

### PR DESCRIPTION
**Pull request by WestleyK**
This followup includes mems comments

### CODE REVIEW

File: pkg/sif/lookup_test.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func TestGetTypeFromLinkedDescr(t *testing.T) {
 		t.Error("multiple partitions found where expecting 1")
```
Comment (ikaneshiro) [inline pkg/sif/lookup_test.go]: Cant comment on the line, but lets change the test name too

File: pkg/sif/lookup_test.go [hunk 2]
```diff
@@ -117,7 +117,7 @@ func TestGetTypeFromLinkedDescr(t *testing.T) {
...
 	}
 
-	fd, snum, err := fimg.GetTypeFromLinkedDescr(parts[0].ID, DataSignature)
+	fd, snum, err := fimg.GetLinkingDescriptorsByType(parts[0].ID, DataSignature)
 	if err != nil {
 		t.Error("fimg.GetFromLinkedDescr(parts[0].ID): should have found descriptor:", err)
```
Comment (ikaneshiro) [inline pkg/sif/lookup_test.go]: And this error message

File: pkg/sif/lookup_test.go [hunk 3]
```diff
@@ -144,7 +144,7 @@ func TestGetTypeFromLinkedDescr(t *testing.T) {
 		t.Error("LoadContainer(testdata/testcontainer1.sif, true):", err)
 	}
 
-	fd, snum, err = fimg.GetTypeFromLinkedDescr(parts[0].ID, DataSignature)
+	fd, snum, err = fimg.GetLinkingDescriptorsByType(parts[0].ID, DataSignature)
 	if err == nil {
 		t.Error("fimg.GetFromLinkedDescr(parts[0].ID): unexpected signature partition: ", err)
```
Comment (ikaneshiro) [inline pkg/sif/lookup_test.go]: here too

File: pkg/sif/lookup_test.go (hunk 1 above)
Comment (WestleyK) [reply pkg/sif/lookup_test.go]: Oh yeah, right :+1:

**Comment by WestleyK** [comment]
>WestleyK dismissed mem’s stale review via 10fa602 17 seconds ago

What?!? I did not?!

**Comment by WestleyK** [comment]
Sorry @mem, can you try that again? _maybe this happen when I forced pushed while your where reviewing?_

**Comment by mem** [comment]
> > WestleyK dismissed mem’s stale review via [10fa602](https://github.com/sylabs/sif/commit/10fa602c9dda54cb32a8780d5127241424fe63fc) 17 seconds ago
> 
> What?!? I did not?!

Did too! :rofl:

**Pull request merged**